src/platform/device.gni

```diff
@@ -103,7 +103,8 @@ declare_args() {
 
   # Enable Subscription persistence / resumption for CI and supported platforms
```

Comment:
It's too bad this is still called persisted subscriptions, when  it's not really persisting subscriptions (i.e. the publisher tries to reconnect with CASE, but without special support by client (which many/most don't and it's not specified how) the subscription is not actually resumed, and may likely have terminated on that side. Yes, you may get CASE again, and it may help latency, but it's not resuming the actual sub in practice.

Reply:
@tcarmelveilleux Fwiw, the subscription _is_ in fact resumed without anything special on the client side as long as the server did not miss a max-interval checkin while rebooting.

Reply:
now i'm not sure if i should open a ticket or not...

Reply:
"resumed" as in the client doesn't think the subscription ever went away.  There's obviously nontrivial resumption work here happening on the _server_ side.